util.go

```diff
@@ -98,6 +100,19 @@ func getUsageCGroup() (float64, float64, int, int, error) {
 	return cpuPercent, memPercent, gNum, tNum, nil
 }
 
+func getCGroupMemoryLimit() (uint64, error) {
+	usage, err := readUint(cgroupMemLimitPath)
+	if err != nil {
+		return 0, err
+	}
+	machineMemory, err := mem_util.VirtualMemoryWithContext(context.TODO())
```

Comment:
possibly it‘s more readable to call `VirtualMemory()` .

Reply:
good idea